### ISSUE TITLE
Fix problem with loading startup page after attempting login with fitbit oauth

### ISSUE THREAD

**Issue by brentm5** [drapergeek/fatty#13]
Logged into the application then authenticated with fitbit, once it was done there was a 500 error when I was returned / every time I log back in.

**Comment by mxie** [comment]
@bigbam505 Is this happening every time you log in or only after you auth through Fitbit?

**Comment by brentm5** [comment]
Everytime I login even if I reset my session

**Comment by brentm5** [comment]
It actually works now, not sure what changed.  might need to look at the error logs

**Comment by mxie** [comment]
Hm, ok. Well this is the error message...not sure how to fix it yet though:

```
2014-01-02T17:22:40.449879+00:00 app[web.1]: Processing by WelcomesController#show as HTML
2014-01-02T17:22:40.475450+00:00 app[web.1]: Completed 500 Internal Server Error in 25ms
2014-01-02T17:22:40.591939+00:00 app[web.1]:   app/models/basic_stat.rb:13:in `sort'
2014-01-02T17:22:40.591939+00:00 app[web.1]:   app/models/basic_stat.rb:13:in `ordered_by_percentage'
2014-01-02T17:22:40.591939+00:00 app[web.1]:   app/controllers/welcomes_controller.rb:3:in `show'
2014-01-02T17:22:40.584875+00:00 app[web.1]: ** [Airbrake] Failure: Net::HTTPUnauthorized
2014-01-02T17:22:40.591939+00:00 app[web.1]:
2014-01-02T17:22:40.591939+00:00 app[web.1]: ArgumentError (comparison of BasicStat with BasicStat failed):
```

**Comment by mxie** [comment]
Fixed in 77415e10015ebc7a496f33bfb2c4251bfd031dff.

The problem was that we weren't pulling the new user's Fitbit information after they registered, and the home page was trying to grab everyone's statistics and there was nothing found for the new user.
